documentation/source/NuRadioReco/pages/times.rst

```diff
@@ -1,52 +1,66 @@
-overview of times
+Overview of times
 =================
-Time delays are introduced by several hardware components. These time delays are often corrected for by folding/unfolding the complex transfer function (for an amp e.g. the measurement of the S12 parameter). The unfolding is typically done in the frequency domain where a convolution becomes a simple multiplication. As a consequence of typically short trace length (<~1000 samples) and because a Fourier transform implies implicitly a periodic signal, a pulse being at the beginning of the trace can end up being at the end of the trace. To avoid this behavior we use the following procedure:
+This page provides an overview of the different times defined in different places in NuRadioMC/NuRadioReco.
+The times corresponding to a voltage or electric field trace can be obtained by the `get_times() <NuRadioReco.framework.base_trace.BaseTrace.get_times>` method,
+which is used by both `electric field <NuRadioReco.framework.electric_field.ElectricField>` and `voltage trace <NuRadioReco.framework.channel.Channel>` classes.
+The start of the trace is also accessible separately as the `trace_start_time <NuRadioReco.framework.base_trace.BaseTrace.get_trace_start_time>`.
+
+Time delays are introduced by several hardware components. These time delays are often corrected for by folding/unfolding the complex transfer function (for an amp e.g. the measurement of the S12 parameter).
+The unfolding is typically done in the frequency domain where a convolution becomes a simple multiplication.
+As a consequence of typically short trace length (<~1000 samples) and because a Fourier transform implies implicitly a periodic signal, a pulse being at the beginning of the trace can end up being at the end of the trace.
+To avoid this behavior we often use the following procedure, through the `NuRadioReco.modules.channelStopFilter` module:
 
 We smoothly filter the first 5% and last 5% of the trace using a Tukey window function. This is a function that goes smoothly from 0 to 1.
-To avoid rollover of the pulse, we add 128ns of zeros to the beginning and end of the trace. Steps 2) and 3) are performed by the channelStopFilter module
-Both electric fields and channels have a trace_start_time variable. The get_times() function will automatically add the trace_start_time to the time array.
+To avoid rollover of the pulse, we add 128ns of zeros to the beginning and end of the trace.
```

Comment:
Is that really true?

Reply:
Yes, but how it is written within the docu suggests that it is the common/default behaviour. Maybe we can add this module to the list of modules which change the timing?

Reply:
It is/was already there. As an aside, it doesn't do what it says here (the trace start time is not adjusted), but I will make a separate bugfix for that.